tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -24,6 +24,34 @@
 pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
 
 VALID: List[Container] = [
+    Container(
+        name="retf_stack_validation_0",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=2),
+            Section.Code(
+                code=Op.PUSH0 * 2 + Op.RETF,
+                code_outputs=2,
+                max_stack_height=2,
+            ),
+        ],
+        expected_bytecode="ef000101000802000200040003040000000080000200020002e30001005f5fe4",
```

Comment:
@marioevz 
the expected_bytecode was a temporary feature, this will require huge efforts of maintenance as eof format changes. consider disabling expected_bytecode verification in most of the EOF tests.

Reply:
I have removed the expected code.